tput smam breaks build

Using latest on Debian Sid.

```
‣  Running finalize script…
‣  Creating tar archive /home/ander/Desktop/mkosi/tools/mkosi.workspace/.mkosi-tmp9zitpbja/staging/image.tar…
‣  /home/ander/Desktop/mkosi/tools/mkosi.output/image size is 1016.1M, consumes 1016.1M.
‣ "tput smam" returned non-zero exit code 1.
make: *** [Makefile:13: build] Error 1
```

`tput` in Debian do not understand `smam`?

```
$ tput smam; echo $?
1
```

Introduced in e651b88.

Ugly workaround:
```
$ ln -s /usr/bin/true ~/.local/bin/tput
```

--- a/mkosi/__main__.py
+++ b/mkosi/__main__.py
@@ -51,8 +51,8 @@ def main() -> None:
         run_verb(args, presets)
     finally:
         if sys.stderr.isatty() and shutil.which("tput"):
-            run(["tput", "cnorm"])
-            run(["tput", "smam"])
+            run(["tput", "cnorm"], check=False)
+            run(["tput", "smam"], check=False)
 
 
 if __name__ == "__main__":

